test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -93,7 +93,7 @@ proc repeatMake(desc, alu, n) {
         s = [i in 0..(r+lineLength)] alu[i % r]: int(8);
 
   for i in 0..n by lineLength {
-    const lo = i % r + 1,
+    const lo = i % r,
```

Comment:
Might want to let Brad know about this change so we can plan for the next re-submission.